docs/source/usage/python.rst

```diff
@@ -390,6 +390,17 @@ Particles
       :return: beam properties with string keywords
       :rtype: dict
 
+   .. py::property:: enable_beam_history
+
+      In situ record and store the beam history for every simulation step (default: ``False``).
```

Comment:
The phrase "beam history" is a bit ambiguous, and it could suggest that the history of every particle is being stored.  Maybe we should specify that what is meant are the reduced diagnostics, etc.